codegen-renderers/src/main/kotlin/io/vrap/codegen/languages/typescript/client/RequestBuilder.kt

```diff
@@ -177,18 +177,11 @@ class RequestBuilder @Inject constructor(
 
 
     fun Resource.imports(moduleName: String): String {
-        return this.methods
+        return this.resources
                 .map {
-                    val requestModuleName = it.tsRequestModuleName((it.toVrapType() as VrapObjectType).`package`)
-                    val relativePath = relativizePaths(moduleName, requestModuleName)
-                    "import { ${it.tsRequestName()} } from '$relativePath'"
+                    val relativePath = relativizePaths(moduleName, it.tsRequestModuleName((this.toVrapType() as VrapObjectType).`package`))
+                    "import { ${it.toRequestBuilderName()} } from '$relativePath'"
                 }.plus(
-                        this.resources
```

Comment:
This change looks interesting. Did this change reduce the number of unused imports?

Reply:
yep , due to the reduced generic parameters we could reduce imports.